libweb3jsonrpc/Debug.cpp

```diff
@@ -92,8 +92,8 @@ Json::Value Debug::debug_traceTransaction(string const& _txHash, Json::Value con
 		Executive e(s, block, t.transactionIndex(), m_eth.blockChain());
 		e.setResultRecipient(er);
 		Json::Value trace = traceTransaction(e, t, _json);
-		ret["gas"] = toHex(t.gas(), HexPrefix::Add);
-		ret["return"] = toHex(er.output, 2, HexPrefix::Add);
+		ret["gas"] = "0x" + toHex(t.gas());
```

Comment:
Perhaps it's better to have a function `toHexWithPrefix` that already allocates the prefix inside the string, saving a copy operation.